src/Helper/ToUtfConverter.php

```diff
@@ -26,8 +26,9 @@ public function convert(string $html, string $contentType = ''): string
         } elseif (preg_match("!.*<meta.*charset=[\"']?[ \t]*([A-Za-z0-9-]{4,})[ \t]*[\"']!mi", $html, $metaContentType) === 1) {
             $encoding = $metaContentType[1];
         }
-        $encoding = strtolower($encoding);
+        $encoding = mb_strtolower($encoding);
         if ($encoding !== '' && in_array($encoding, $this->getSupportedEncodings(), true)) {
+            /** @noinspection CallableParameterUseCaseInTypeContextInspection */
```

Comment:
Can you explain why we get complain from the inspection ;)

Reply:
The mb_convert_encoding method can return an array if an array was supplied as input. In this case, I know that my argument is a string, so the array will never be returned. It would be nice to also check for false, but that's another story.

Reply:
And ahead of your next question, I will say that you cannot simply convert to string in this case.)